go.mod

```diff
@@ -1,14 +1,15 @@
-module github.com/bndr/gojenkins
+module github.com/IvenGe/gojenkins
```

Comment:
Please don't change the module path, thanks

Reply:
I reverted the change